orttraining/orttraining/test/python/orttraining_test_ortmodule_bert_classifier.py

```diff
@@ -312,6 +312,49 @@ def format_time(elapsed):
     # Format as hh:mm:ss
     return str(datetime.timedelta(seconds=elapsed_rounded))
 
+def bert_classifier_log_perf_metrics(args, train_dataloader, perf_metrics_start_time):
+    total_train_steps = 0
+    if args.train_steps > 0:
+        total_train_steps = args.train_steps * args.epochs
+    else:
+        total_train_steps = len(train_dataloader) * args.epochs
+
+    perf_metrics = {}
+    perf_metrics['Model'] = 'orttraining_test_ortmodule_bert_classifier'
+    perf_metrics['BatchId'] = 'na'
```

Comment:
>BatchId [](start = 18, length = 7)

just to clarify, the batchId is populated using datetime, I wonder how this field will be populated. Perhaps you can leverage datetime.datetime.now() again.